openedx2zim/utils.py

```diff
@@ -334,21 +336,17 @@ def get_response_headers(url):
         response_headers = get_response_headers(url)
     except Exception as exc:
         logger.error(f"{url} > Problem with head request\n{exc}\n")
-        return None, None
+        return None, None, None
     else:
-        filetype = response_headers.get("content-type", None)
-        if filetype:
-            filetype = filetype.split("/")[-1]
+        content_type = mimetypes.guess_extension(
+            response_headers.get("content-type", None).partition(";")[0].strip()
```

Comment:
Usually when you use only first part, `split(";", 1)` makes more sense. But it's equivalent here.

Reply:
Okay. Changed that.